site/docs/id/plugins/i18n.md

```diff
@@ -396,10 +400,13 @@ type MyContext = Context & SessionFlavor<SessionData> & I18nFlavor;
 const i18n = new I18n<MyContext>({
   defaultLocale: "en",
   useSession: true, // Menentukan data bahasa user disimpan di session atau tidak
-  directory: "locales", // Muat locale dari direktori `locales`.
+
+  // TIDAK BEKERJA di Deno Deploy
+  directory: "locales",
 });
 
-const bot = new Bot<MyContext>(""); // <-- taruh token bot kamu di sini.
+// Untuk Deno Deploy, pakai cara berikut untuk memuat file terjemahan.
+// await i18n.loadLocalesDir("locales");
 
```

Comment:
Missing here: `const bot = new Bot<MyContext>(""); // <-- put your bot token here` translated

Reply:
I suppose is: `const bot = new Bot<MyContext>(""); // <-- taruh token bot kamu di sini.`

Reply:
Fixed by me, verify before merge

Reply:
See this discussion https://github.com/grammyjs/website/pull/515#discussion_r981603054